compiler/rustc_borrowck/src/type_check/liveness/trace.rs

```diff
@@ -218,21 +216,38 @@ impl<'me, 'typeck, 'flow, 'tcx> LivenessResults<'me, 'typeck, 'flow, 'tcx> {
     ///
     /// Add facts for all locals with free regions, since regions may outlive
     /// the function body only at certain nodes in the CFG.
-    fn add_extra_drop_facts(
-        &mut self,
-        drop_used: Vec<(Local, Location)>,
-        relevant_live_locals: FxIndexSet<Local>,
-    ) {
+    fn add_extra_drop_facts(&mut self, relevant_live_locals: &[Local]) -> Option<()> {
+        let drop_used = self
+            .cx
+            .typeck
+            .borrowck_context
+            .all_facts
+            .as_ref()
+            .map(|facts| facts.var_dropped_at.clone())?;
```

Comment:
drive-by question: how big is this vec?

Reply:
When I wrote my master’s thesis drop-uses weren’t super common so I’d say small ish, but I’d like to remove this clone if possible. 

Of course I had a dog walk idea how to do that just in time for it to get merged before I’d implemented it but I’ll look into it next time I need something to pass the time

Reply:
I should add that the clone happened in the original version too, except not as a clone but as a for each insert.